### PR TITLE
Validate errors show on toasts

### PR DESCRIPTION
**Pull request by joaojusto**
screen shot 2015-01-13 at 15 53 27

### CODE REVIEW

File: app/assets/javascripts/app/discussions/edit.js [hunk 1]
```diff
@@ -19,6 +20,30 @@
...
+        showToast('Please fill the required fields');
+      }
+    }
+
+    function isValid() {
+      if ($scope.form.$valid)
+        return true;
+      else
+        return false;
+    }
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/edit.js]: `return $scope.form.$valid`

File: app/assets/javascripts/app/discussions/new.js [hunk 2]
```diff
@@ -5,13 +5,38 @@
...
+    }
+
+    function showToast(message) {
+      $mdToast.show(
+        $mdToast.simple()
+          .content(message)
+          .position('bottom right')
+          .hideDelay(2000)
+      );
+    }
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/new.js]: Is this copy paste from the previous file?

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/new.js]: Yes are the same functions

**Comment by joaojusto** [comment]
@ronaldofs @gabrielpoca

File: app/assets/javascripts/app/discussions/discussion_form.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  'use strict';
+
+  angular
+    .module('council.discussions')
+    .controller('DiscussionFormCtrl', DiscussionFormCtrl);
+
+  function DiscussionFormCtrl(Discussion, $state, $mdToast, $scope, discussion) {
+    var ctrl = this;
+
+    $scope.next = next;
```
Comment (ronaldofs) [inline app/assets/javascripts/app/discussions/discussion_form.js]: remove $scope

File: app/assets/javascripts/app/discussions/discussion_form.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      );
+    }
+
+    function createDiscussion(discussion) {
+      Discussion.create(discussion).then(onDiscussionSave);
+    }
+
+    function updateDiscussion(discussion) {
+      Discussion.create(discussion).then(onDiscussionSave);
+    }
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/discussion_form.js]: Do we really need to the methods? Can't we rename it to something like `submitDiscussion` or `saveDiscussion`?

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/discussion_form.js]: Yes we can

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**